Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -625,6 +625,9 @@
 workflows[1362.18] = ['', ['VBFHToZZTo4Nu_M125_Pow_py8_Evt_13UP18','DIGIUP18','RECOUP18','HARVESTUP18']]
 workflows[1363.18] = ['', ['VBFHToBB_M125_Pow_py8_Evt_13UP18','DIGIUP18','RECOUP18','HARVESTUP18']]
 
+#2018 workflows starting from gridpacks LHE generation with multiple concurrent lumi sections
+workflows[1361.181] = ['', ['GluGluHToZZTo4L_M125_Pow_py8_Evt_13UP18ml','DIGIUP18ml','RECOUP18ml','HARVESTUP18']]
+
 # fullSim 13TeV normal workflows starting from pLHE
 workflows[1370] = ['', ['GluGluHToGG_M125_Pow_MINLO_NNLOPS_py8_13','Hadronizer_TuneCUETP8M1_13TeV_powhegEmissionVeto_2p_HToGG_M125_13','DIGIUP15','RECOUP15','HARVESTUP15']]
```

Comment:
Hi @fabiocos , i'm not sure if understood correctly but from https://github.com/cms-sw/cmssw/issues/25708 @srimanob concerned about pLHE+GS production. We have workflows 1370,1370.17,1370.18 in our standard relval list which start from external source LHE (or pLHE). The one you added will produce LHE first through gridpack and then get into GS step, similar to our wmLHEGS in production. Do we plan to expand these settings in relval standard list? If so, i would suggest to work on top of 1370,1370.17,1370.18.

Reply:
@chayanit my reference to #25708 was an example of issue that happens at the boundary between lumi sections. This is a minimal proposal to add 1 wf that tests systematically the crossing of such a boundary in the whole chain. This can be expanded of course, but there is no big advantage in my view to add just clones of this for older setups of this workflow.